src/main/java/org/acme/infinispanclient/model/HPCharacter.java

```diff
@@ -0,0 +1,70 @@
+package org.acme.infinispanclient.model;
+
+import java.util.Objects;
+
+/**
+ * Harry Potter saga character
+ */
+public class HPCharacter {
+   private final int id;
+   private final String name;
+   private final String bio;
+   private final int type;
```

Comment:
Something to clarify this type would be helpful. I am guessing 1 is Teacher and 2 is Student? 4 must be Muggle, but what else is there? Can we not use an ENUM instead?

Reply:
Did you not like the enum idea? It seems like it would be a lot clearer to me.

Reply:
yes, it was my idea at first, I just wanted to go fast at a point and I forgot after. thanks for the heads up, I'm going to read all the outdated stuff since it's hiding some comments.
I think this outdated thing is because I renamed packages so github interfaces hides them.
need to tidy my e-mail too btw